spec/fixtures/collection/dummy.xml

```diff
@@ -0,0 +1,146 @@
+<title language="en" format="text/plain" type="title-main">Dummy document</title>
+<title language="fr" format="text/plain" type="main">Dummy document</title>
+<title language="fr" format="text/plain" type="title-main">Dummy document</title>
+<docidentifier type="ISO">ISO 17302:2016</docidentifier>
+<docidentifier type="iso-with-lang">ISO 17302:2016(E)</docidentifier>
+<docidentifier type="iso-reference">ISO 17302:2016(E)</docidentifier>
+<docnumber>17302</docnumber>
+<contributor>
+<role type="author"/>
+<organization>
+<name>International Organization for Standardization</name>
+<abbreviation>ISO</abbreviation>
+</organization>
+</contributor>
+<contributor>
+<role type="publisher"/>
+<organization>
+<name>International Organization for Standardization</name>
+<abbreviation>ISO</abbreviation>
+</organization>
+</contributor>
+
+<language>en</language>
+<script>Latn</script>
+<status>
+<stage abbreviation="IS">60</stage>
+<substage>60</substage>
+</status>
+<copyright>
+<from>2016</from>
+<owner>
+<organization>
+<name>International Organization for Standardization</name>
+<abbreviation>ISO</abbreviation>
+</organization>
+</owner>
+</copyright>
+<ext>
+<doctype>international-standard</doctype>
+<editorialgroup>
+<technical-committee number="34">Food products</technical-committee>
+<subcommittee number="4">Cereals and pulses</subcommittee>
+<workgroup number="4" type="WG">Amylose in rice</workgroup>
+<secretariat>SAC</secretariat>
+</editorialgroup>
+<ics>
+<code>67.060</code>
+</ics>
+<structuredidentifier>
+<project-number>ISO 17302</project-number>
+</structuredidentifier>
+<stagename>International standard</stagename>
+</ext>
+</bibdata>
+<boilerplate>
+  <copyright-statement>
+    <clause>
+    <p id="boilerplate-year">
+    © ISO 2016
+  </p>
+
+  <p id="boilerplate-message">
+    All rights
+reserved. Unless otherwise specified, no part of this publication may be
+reproduced or utilized otherwise in any form or by any means, electronic or
+mechanical, including photocopying, or posting on the internet or an intranet,
+without prior written permission. Permission can be requested from either ISO
+at the address below or ISO’s member body in the country of the requester.
+  </p>
+
+  <p id="boilerplate-address" align="left">
+    ISO copyright office<br/>
+    Ch. de Blandonnet 8 • CP 401<br/>
+    CH-1214 Vernier, Geneva, Switzerland<br/>
+    Tel.  + 41 22 749 01 11<br/>
+    Fax  + 41 22 749 09 47<br/>
+    Email: copyright@iso.org<br/>
+    Website: www.iso.org
+  </p>
+    <p id="boilerplate-place">
+    Published in Switzerland
+  </p>
+</clause>
+  </copyright-statement>
+
+  
+</boilerplate>
+<sections><clause id="_scope" obligation="normative"><title>Scope</title><p id="_9f7c3e34-6854-4fa6-bb3b-e967ff9f5bf2">This is a dummy document.</p>
+<figure id="_b4013f8f-ee40-4186-a5ed-3a5103f199e1">
+<pre id="_0855e974-d21b-48ad-8627-0c53719b8dcf">~/Documents/Arbeit/upwork/ribose/isodoc/lib/isodoc/function[master] $ ls -l
```

Comment:
Let's replace this source code content with some "source code" instead of this.

Reply:
done, but it needs to replace `<`, `>`, `&` with `&lt;`, `&gt;`, `&amp;`